src/router/router.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express");
```

Comment:
Github won't open up the echoesPI.json file. Maybe it should be ignored? 🤔 (sorry i couldn't comment on that file directly)

Reply:
Is required for use on the server. When time allows, maybe installation scripts or even a database can be added.